.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -2,25 +2,52 @@
 name: Feature request
 about: Suggest an idea for this project
 title: ''
-labels: feature
+labels: 'type: feature'
```

Comment:
Remove this please

Reply:
No tags for features / bugs without any discussion or reviews by team members first